api/src/handlers/cairo_version.rs

```diff
@@ -11,7 +12,7 @@ use tracing::{error, info, instrument};
 #[get("/cairo_version")]
 pub async fn cairo_version() -> String {
     info!("/cairo_version");
-    do_cairo_version().unwrap_or_else(|e| e)
+    do_cairo_version().unwrap_or_else(|e| format!("Failed to get cairo version: {:?}", e))
```

Comment:
Just curious, all other errors are specified with an enum, why not this too?

Reply:
Because we return string in handlers, we just format our returned enum, but, I think, we can return some JSON error